### PR TITLE
Milestone 2 - Work and About Me section

### PR DESCRIPTION
**Pull request by t-bello7**
- add about work and about me section
- add new utility classes and fonts (centre, mg-inline, ff-inia)
- utilize grid layout for the work section container

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
...
+            <p class="ff-inter"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+            <ul class="tag tag-feature mg-1 ff-inter flex">
+                <li > css </li>
+                <li > html </li>
+                <li> bootstrap </li>
+                <li> Ruby </li>
+            </ul>
+            <button class="btn" type="button"> See Project </button>
+        </div>
+        <div class="flex project-container">
```
Comment (GzimAsani) [inline index.html:47]: - [ ] The main container of the projects should have  a `display:grid`

File: index.html [hunk 2]
```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
...
+            <p class="mg-1 ff-inter text-light">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button class="btn" type="button"> Get my Resume</button>
+        </div>
+        <div class=" flex skill">
+            <h3 class="text-light ff-inai"> <span class="skill-icon"></span>   Language</h3>
+            <ul class="center ff-inter">
+                <li class="text-light mg-1">Javascript</li>
+                <li class="text-light mg-1">Ruby</li>
+                <li class="text-light mg-1">HTML</li>
+                <li class="text-light mg-1">CSS</li>
```
Comment (GzimAsani) [inline index.html]: - [ ] _The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)_. Please add the `a` tags and the list of icons with your GitHub and etc sites

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @t-bello7   
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
 - _Check the comments below :arrow_double_down:_ 




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (t-bello7) [reply index.html:47]: Screenshot from 2022-04-23 19-38-13

@GzimAsani Thank you for the review. I am of the mindset that the display: grid container is for both the feature project and other projects.

File: index.html (hunk 2 above)
Comment (t-bello7) [reply index.html]: Thank you @GzimAsani for the review. Am I to add the link tag to official websites of the various languages and frameworks ?

Comment (GzimAsani) [reply index.html]: @t-bello7 Yes

File: index.html [hunk 3]
```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
+        <h2 class="ff-crete mg-8 center">My Recent Works</h2>
```
Comment (prolajumokeoni) [inline index.html:34]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_69  | Screenshot_70

- [ ] Kindly adjust the spacing between the my recent work header, please reduce the spacing and make it look mere like the Figma design

File: index.html [hunk 4]
```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
...
+        <div class="featured">
+            <img src="assets/img/img-placeholder.png" alt="">
+            <h3 class="mg-1 ff-crete" >Multi-Post Stories</h3>
+            <p class="ff-inter"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+            <ul class="tag tag-feature mg-1 ff-inter flex">
+                <li> <a href="#"> css </a> </li>
+                <li> <a href="#"> html </a> </li>
+                <li> <a href="#"> bootstrap </a> </li>
+                <li> <a href="#"> Ruby </a> </li>
```
Comment (prolajumokeoni) [inline index.html:43]: - [ ] Kindly ensure that all the li tags are on a single row, please check the screenshot above and make your design look more like the Figma design 👍🏿

File: index.html [hunk 5]
```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
```
Comment (prolajumokeoni) [inline index.html:33]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_72  | Screenshot_71

- [ ] Kindly increase the width of each of the cards so it looks exactly like the Figma design 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @t-bello7, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design 



## Required Changes ♻️
_Check the comments under the review._
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @t-bello7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**